### ISSUE TITLE
Stream serialization exmple doesn't work with Python 3.5.1

### ISSUE THREAD

**Issue by utgarda** [vsergeev/u-msgpack-python#16]
```console
[etsvigun@Jotunheim ~]$ python
Python 3.5.1 (default, Dec  7 2015, 12:58:09) 
[GCC 5.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import umsgpack
>>> f = open('test.bin', 'w')
>>> umsgpack.pack({u"compact": True, u"schema":0}, f)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.5/site-packages/umsgpack.py", line 428, in _pack3
    _pack_map(obj, fp)
  File "/usr/lib/python3.5/site-packages/umsgpack.py", line 323, in _pack_map
    fp.write(struct.pack("B", 0x80 | len(obj)))
TypeError: write() argument must be str, not bytes
```

**Comment by vsergeev** [comment]
Thanks for bringing this to my attention. I'll fix the documentation.

The file should be opened with binary mode:

```
>>> import umsgpack
>>> f = open('test.bin', 'wb')
>>> umsgpack.pack({u"compact": True, u"schema":0}, f)
>>> 
```

**Comment by vsergeev** [comment]
Fixed in 073dfd2.

**Comment by utgarda** [comment]
Thanks @vsergeev !
